src/dlgTriggerEditor.cpp

```diff
@@ -6832,6 +7144,113 @@ void dlgTriggerEditor::slot_export()
     };
 }
 
+void dlgTriggerEditor::slot_copy_xml()
+{
+    switch( mCurrentView )
+    {
+    case cmTriggerView:
+        exportTriggerToClipboard( );
+        break;
+    case cmTimerView:
+        exportTimerToClipboard( );
+        break;
+    case cmAliasView:
+        exportAliasToClipboard( );
+        break;
+    case cmScriptView:
+        exportScriptToClipboard( );
+        break;
+    case cmActionView:
+        exportActionToClipboard( );
+        break;
+    case cmKeysView:
+        exportKeyToClipboard( );
+        break;
+    };
+}
+
+void dlgTriggerEditor::slot_paste_xml()
+{
+    switch (mCurrentView) {
+    case cmTriggerView:
+        saveTrigger();
+        break;
+    case cmTimerView:
+        saveTimer();
+        break;
+    case cmAliasView:
+        saveAlias();
+        break;
+    case cmScriptView:
+        saveScript();
+        break;
+    case cmActionView:
+        saveAction();
+        break;
+    case cmKeysView:
+        saveKey();
+        break;
+    };
+
+    QString profileName = mpHost->getName();
+    QString login = mpHost->getLogin();
+    QString pass = mpHost->getPass();
+
+    treeWidget_triggers->clear();
+    treeWidget_aliases->clear();
+    treeWidget_actions->clear();
+    treeWidget_timers->clear();
+    treeWidget_keys->clear();
+    treeWidget_scripts->clear();
+
+    XMLimport reader(mpHost);
+    int importedItemType, importedItemID;
+    std::tie(importedItemType, importedItemID) = reader.importFromClipboard();
+
+    mpHost->setName(profileName);
+    mpHost->setLogin(login);
+    mpHost->setPass(pass);
```

Comment:
Same as above: This was in the code I hijacked from `void dlgTriggerEditor::slot_import()`. Seems to run fine without, so it's gone now.